### PR TITLE
Feature/segmented chip

### PR DESCRIPTION
**Pull request by azeemchauhan**
We are adding following new component in kirby
- Segmented Chip control
- simple chip
- update in kirby cookbook

### CODE REVIEW

File: src/kirby/components/chip/chip.component.tns.scss [hunk 1]
```diff
@@ -1,10 +1,13 @@
 @import '../../scss/native-imports';
+$min-card-width: 50;
+$chip-active-color:#cfd4d3;
```
Comment (gags88) [inline src/kirby/components/chip/chip.component.tns.scss]: Please consider using variables. Do not use hex codes

File: src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+   <div class="scrollable">
```
Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.html]: Fix type in file name

File: src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+:host {
+    /* make area with scroll hidden */
+    overflow: hidden;
+  .scrollable {
+    display: flex;
+    flex-wrap: nowrap;
+    overflow-x: auto;
+    justify-content: center;
+    /* add to be able to scroll */
```
Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.scss]: I dont understand this?

File: src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.tns.html [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<ScrollView orientation="horizontal">
+    <WrapLayout orientation="horizontal">
+        <ng-template ngFor let-item [ngForOf]="items">
```
Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.tns.html]: Don't use `ng-template` - use `ng-container`

Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.tns.html]: No need for wrapping - just move *ngFor loop to `StackLayout` element

File: src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.tns.html [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+            <StackLayout (tap)="segmentedChipSelect(item)">
+                <ng-container *ngTemplateOutlet="segmentedChipTemplate; context: { $implicit: item}"></ng-container>
```
Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.tns.html]: Omit this and render chips from items array instead

File: src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+@Component({
+  selector: 'kirby-segmented-chip-control',
+  templateUrl: './segmentes-chip-control.component.html',
+  styleUrls: ['./segmentes-chip-control.component.scss'],
+})
+class SegmentedChipControlComponent implements OnInit {
+  @Input() items: SegmentChip[];
+  @Output() chipSelect = new EventEmitter<SegmentChip>();
+  @ContentChild(SegmentedChipDirective, { read: TemplateRef }) segmentedChipTemplate;
```
Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.ts]: Omit this and render chips from items array instead

File: src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.ts [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  EventEmitter,
+} from '@angular/core';
+
+interface SegmentChip {
+  index: number;
+  title: string;
+}
+
+@Directive({
+  selector: '[kirbySegmentedChip]',
```
Comment (jakobe) [inline src/kirby/components/segment/segmentes-chip-control/segmentes-chip-control.component.ts]: Omit this and render chips from items array instead

File: src/app/examples/chip-example/chip-example.component.html [hunk 8]
```diff
@@ -1,2 +1,2 @@
-<kirby-chip [title]="'kirby'"></kirby-chip>
-<kirby-chip [title]="'kirby'" [isActive]="true"></kirby-chip>
+<kirby-chip label="kirby"></kirby-chip>
```
Comment (jakobe) [inline src/app/examples/chip-example/chip-example.component.html]: Rename to `text` as we use for buttons, badge etc.
```suggestion
<kirby-chip text="kirby"></kirby-chip>
```

File: src/app/examples/chip-example/chip-example.component.html [hunk 9]
```diff
@@ -0,0 +1,2 @@
...
+<kirby-chip label="kirby" isSelected="true"></kirby-chip>
```
Comment (jakobe) [inline src/app/examples/chip-example/chip-example.component.html]: Rename to `text` as we use for buttons, badge etc.
```suggestion
<kirby-chip text="kirby" isSelected="true"></kirby-chip>
```

File: src/app/examples/chip-example/chip-example.component.tns.html [hunk 10]
```diff
@@ -0,0 +1,4 @@
+<WrapLayout orientation="horizontal" class="chip-example">
+  <kirby-chip label="kirby"></kirby-chip>
```
Comment (jakobe) [inline src/app/examples/chip-example/chip-example.component.tns.html]: Rename to `text` as we use for buttons, badge etc.
```suggestion
  <kirby-chip text="kirby"></kirby-chip>
```

File: src/app/examples/chip-example/chip-example.component.tns.html [hunk 11]
```diff
@@ -0,0 +1,4 @@
...
+  <kirby-chip label="kirby" isSelected="true"></kirby-chip>
```
Comment (jakobe) [inline src/app/examples/chip-example/chip-example.component.tns.html]: Rename to `text` as we use for buttons, badge etc.
```suggestion
  <kirby-chip text="kirby" isSelected="true"></kirby-chip>
```

File: src/app/examples/examples.common.ts [hunk 12]
```diff
@@ -8,6 +8,8 @@ import { ListLoadOnDemandExampleComponent } from './list/load-on-demand/list-loa
 import { ChartExampleComponent } from './chart-example/chart-example.component';
 import { FontsExampleComponent } from './fonts-example/fonts-example.component';
 import { SpinnerExampleComponent } from './spinner-example/spinner-example.component';
+import { ChipExampleComponent } from './chip-example/chip-example.component';
+import { SegmentExampleComponent } from './segment-example/segment-example.component';
```
Comment (jakobe) [inline src/app/examples/examples.common.ts]:
```suggestion
import { SegmentedChipControlExampleComponent } from './segmented-chip-control-example/segmented-chip-control-example.component';
```

File: src/app/examples/examples.common.ts [hunk 13]
```diff
@@ -22,6 +24,8 @@ export const COMPONENT_DECLARATIONS: any[] = [
   ChartExampleComponent,
   FontsExampleComponent,
   SpinnerExampleComponent,
+  SegmentExampleComponent,
```
Comment (jakobe) [inline src/app/examples/examples.common.ts]:
```suggestion
  SegmentedChipControlExampleComponent,
```

File: src/app/examples/examples.routes.ts [hunk 14]
```diff
@@ -9,6 +9,8 @@ import { ListExampleComponent } from './list/list-example.component';
 import { ChartExampleComponent } from './chart-example/chart-example.component';
 import { FontsExampleComponent } from './fonts-example/fonts-example.component';
 import { SpinnerExampleComponent } from './spinner-example/spinner-example.component';
+import { SegmentExampleComponent } from './segment-example/segment-example.component';
```
Comment (jakobe) [inline src/app/examples/examples.routes.ts]:
```suggestion
import { SegmentedChipControlExampleComponent } from './segmented-chip-control-example/segmented-chip-control-example.component';
```

File: src/app/examples/examples.routes.ts [hunk 15]
```diff
@@ -53,6 +59,10 @@ export const routes: Routes = [
     path: 'spinner',
     component: SpinnerExampleComponent,
   },
+  {
+    path: 'segment',
+    component: SegmentExampleComponent,
```
Comment (jakobe) [inline src/app/examples/examples.routes.ts]:
```suggestion
    component: SegmentedChipControlExampleComponent,
```

File: src/app/examples/examples.routes.ts [hunk 16]
```diff
@@ -53,6 +59,10 @@ export const routes: Routes = [
     path: 'spinner',
     component: SpinnerExampleComponent,
   },
+  {
+    path: 'segment',
```
Comment (jakobe) [inline src/app/examples/examples.routes.ts]:
```suggestion
    path: 'segmented-chip-control',
```

File: src/app/examples/segment-example/segment-example.component.html [hunk 17]
```diff
@@ -0,0 +1,5 @@
+<kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)" [selectedItem]="selectedItem"></kirby-segmented-chip-control>
```
Comment (jakobe) [inline src/app/examples/segment-example/segment-example.component.html]: Follow [Angular Style Guide 05-16](https://angular.io/guide/styleguide#style-05-16): "Do name event handler methods with the prefix on followed by the event name."
```suggestion
<kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)" [selectedItem]="selectedItem"></kirby-segmented-chip-control>
```

File: src/app/examples/segment-example/segment-example.component.html [hunk 18]
```diff
@@ -0,0 +1,5 @@
...
+
+<div class="segment-data" *ngIf="selectedItem">
+        Content of tab {{selectedItem}}
```
Comment (jakobe) [inline src/app/examples/segment-example/segment-example.component.html]:
```suggestion
        Content of chip: {{selectedItem}}
```

File: src/app/examples/segment-example/segment-example.component.tns.html [hunk 19]
```diff
@@ -0,0 +1,7 @@
+<StackLayout class="segment-example">
+        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)" [selectedItem]="selectedItem">
```
Comment (jakobe) [inline src/app/examples/segment-example/segment-example.component.tns.html]: Follow [Angular Style Guide 05-16](https://angular.io/guide/styleguide#style-05-16): "Do name event handler methods with the prefix on followed by the event name."
```suggestion
        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)" [selectedItem]="selectedItem">
```

File: src/app/examples/segment-example/segment-example.component.tns.html [hunk 20]
```diff
@@ -0,0 +1,7 @@
...
+                </kirby-segmented-chip-control>
+        <StackLayout class="segment-data" *ngIf="selectedItem">
+                <Label text="Content of tab {{selectedItem}}" textWrap="true"></Label>
```
Comment (jakobe) [inline src/app/examples/segment-example/segment-example.component.tns.html]:
```suggestion
                <Label text="Content of chip: {{selectedItem}}" textWrap="true"></Label>
```

File: src/app/examples/segment-example/segment-example.component.ts [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+@Component({
+  selector: 'kirby-segment-example',
+  templateUrl: './segment-example.component.html',
+  styleUrls: ['./segment-example.component.scss'],
+})
+export class SegmentExampleComponent implements OnInit {
+  constructor() {}
+  items: string[] = ['chip-1', 'chip-2', 'chip-3', 'chip-4', 'chip-5', 'chip-6'];
+  selectedItem: string = 'chip-1';
+  onChipSelected(chip: string): void {
```
Comment (jakobe) [inline src/app/examples/segment-example/segment-example.component.ts]: Follow [Angular Style Guide 05-16](https://angular.io/guide/styleguide#style-05-16): "Do name event handler methods with the prefix on followed by the event name."
```suggestion
  onChipSelect(chip: string): void {
```

File: src/app/examples/segment-example/segment-example.component.ts [hunk 22]
```diff
@@ -0,0 +1,16 @@
...
+
+@Component({
+  selector: 'kirby-segment-example',
+  templateUrl: './segment-example.component.html',
+  styleUrls: ['./segment-example.component.scss'],
+})
+export class SegmentExampleComponent implements OnInit {
+  constructor() {}
+  items: string[] = ['chip-1', 'chip-2', 'chip-3', 'chip-4', 'chip-5', 'chip-6'];
+  selectedItem: string = 'chip-1';
```
Comment (jakobe) [inline src/app/examples/segment-example/segment-example.component.ts]:
```suggestion
  selectedItem: string = this.items[0];
```

File: src/app/intro/intro.component.tns.html [hunk 23]
```diff
@@ -11,6 +11,8 @@
     <Button kirby-button text="Fonts" [nsRouterLink]="['/examples/fonts']" ></Button>
     <Button kirby-button text="Charts" [nsRouterLink]="['/examples/chart']" ></Button>
     <Button kirby-button text="Spinner" [nsRouterLink]="['/examples/spinner']" ></Button>
+    <Button kirby-button text="Segment" [nsRouterLink]="['/examples/segment']" ></Button>
```
Comment (jakobe) [inline src/app/intro/intro.component.tns.html]:
```suggestion
    <Button kirby-button text="Segmented Chip" [nsRouterLink]="['/examples/segment-chip-control']" ></Button>
```

File: src/app/page/side-nav/side-nav.component.html [hunk 24]
```diff
@@ -24,6 +24,7 @@
       <li><a routerLink="showcase/button" [routerLinkActive]="'is-selected'" class="side-nav__item">Button</a></li>
       <li><a routerLink="showcase/floating-action-button" [routerLinkActive]="'is-selected'" class="side-nav__item">Floating Action Button</a></li>
       <li><a routerLink="showcase/card" [routerLinkActive]="'is-selected'" class="side-nav__item">Card</a></li>
+      <li><a routerLink="showcase/segment" [routerLinkActive]="'is-selected'" class="side-nav__item">Segment</a></li>
```
Comment (jakobe) [inline src/app/page/side-nav/side-nav.component.html]:
```suggestion
      <li><a routerLink="showcase/segmented-chip-control" [routerLinkActive]="'is-selected'" class="side-nav__item">Segmented Chip Control</a></li>
```

File: src/app/showcase/chip-showcase/chip-showcase.component.html [hunk 25]
```diff
@@ -0,0 +1,24 @@
...
+    <h4>Properties:</h4>
+    <table class="cookbook-properties">
+      <thead>
+        <th>Name</th>
+        <th>Description</th>
+        <th>Input values</th>
+      </thead>
+      <tbody>
+        <tr>
+          <td><code>label</code></td>
```
Comment (jakobe) [inline src/app/showcase/chip-showcase/chip-showcase.component.html]:
```suggestion
          <td><code>text</code></td>
```

File: src/app/showcase/chip-showcase/chip-showcase.component.html [hunk 26]
```diff
@@ -0,0 +1,24 @@
...
+          <td>use this as Label.</td>
```
Comment (jakobe) [inline src/app/showcase/chip-showcase/chip-showcase.component.html]:
```suggestion
          <td>The text displayed inside the chip.</td>
```

File: src/app/showcase/segment-showcase/segment-showcase.component.html [hunk 27]
```diff
@@ -0,0 +1,24 @@
...
+      </thead>
+      <tbody>
+         <tr>
+          <td><code>items</code></td>
+          <td>Array of items.</td>
+          <td><code>Array of type &#123;title, index&#125;</code></td>
+        </tr>  
+        <tr>
+          <td><code>chipSelect</code></td>
+          <td>It will give selected chip.</td>
```
Comment (jakobe) [inline src/app/showcase/segment-showcase/segment-showcase.component.html]:
```suggestion
          <td>Emitted when the selected chip has changed.</td>
```

File: src/app/showcase/segment-showcase/segment-showcase.component.html [hunk 28]
```diff
@@ -0,0 +1,24 @@
...
+          <td><code>function</code></td>
```
Comment (jakobe) [inline src/app/showcase/segment-showcase/segment-showcase.component.html]:
```suggestion
          <td><code>event</code></td>
```

File: src/kirby/components/chip/chip.component.html [hunk 29]
```diff
@@ -0,0 +1,3 @@
+<div class="chip" [class.selected]="isSelected">
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.html]: No need to wrap in additional layer, just use `:host`styling:
```css
:host[isSelected] {...}
```
```suggestion
```

File: src/kirby/components/chip/chip.component.scss [hunk 30]
```diff
@@ -0,0 +1,17 @@
+@import "../../scss/web-imports";
+$min-chip-width: 50px;
+.chip {
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.scss]: No need to wrap in additional layer, just use `:host` styling:
```suggestion
:host {
```

File: src/kirby/components/chip/chip.component.scss [hunk 31]
```diff
@@ -0,0 +1,17 @@
...
+  white-space:nowrap;
+  padding: 0 25px;
+  height: 30px;
+  font-size: font-size('s');
+  line-height: line-height('xl');
+  border-radius: 25px;
+  margin-right: 5px;
+  min-width: $min-chip-width;
+  cursor: pointer;
+  &:hover, &.selected{
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.scss]:
```suggestion
  &:hover, &[isSelected] {
```

File: src/kirby/components/chip/chip.component.ts [hunk 32]
```diff
@@ -0,0 +1,13 @@
+import { Component, ViewEncapsulation, Input } from '@angular/core';
+
+@Component({
+  selector: 'kirby-chip',
+  templateUrl: './chip.component.html',
+  styleUrls: ['./chip.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class ChipComponent {
+  @Input() label: string;
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.ts]:
```suggestion
  @Input() text: string;
```

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html [hunk 33]
```diff
@@ -0,0 +1,5 @@
+<div class="scrollable">
+    <div *ngFor="let item of items" (click)="segmentedChipSelect(item)">
+        <kirby-chip [label]="item" [isSelected]="item.toLowerCase() === selectedItem.toLowerCase()"></kirby-chip>
```
Comment (jakobe) [inline src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html]:
```suggestion
        <kirby-chip [text]="item" [isSelected]="item.toLowerCase() === selectedItem.toLowerCase()"></kirby-chip>
```

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html [hunk 34]
```diff
@@ -0,0 +1,7 @@
+<ScrollView orientation="horizontal">
+    <WrapLayout orientation="horizontal">
+        <StackLayout *ngFor="let item of items" (tap)="segmentedChipSelect(item)">
+            <kirby-chip [label]="item" [isSelected]="item === selectedItem"></kirby-chip>
```
Comment (jakobe) [inline src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]:
```suggestion
            <kirby-chip [text]="item" [isSelected]="item.toLowerCase() === selectedItem.toLowerCase()"></kirby-chip>
```

File: src/app/showcase/segmented-chip-control-showcase/segmented-chip-control-showcase.component.html [hunk 35]
```diff
@@ -0,0 +1,29 @@
...
+    <thead>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Input values</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>items</code></td>
+        <td>Array of items</td>
+        <td><code>Array of string</code></td>
```
Comment (jakobe) [inline src/app/showcase/segmented-chip-control-showcase/segmented-chip-control-showcase.component.html]:
```suggestion
        <td><code>string[]</code></td>
```

File: src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.html [hunk 36]
```diff
@@ -0,0 +1,6 @@
+<kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)" [selectedItem]="selectedItem">
```
Comment (jakobe) [inline src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.html]: Use naming convention:
```suggestion
<kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)" [selectedItem]="selectedItem">
```

File: src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.tns.html [hunk 37]
```diff
@@ -0,0 +1,8 @@
+<StackLayout class="segment-example">
+        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelected($event)"
```
Comment (jakobe) [inline src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.tns.html]: Use naming convention:
```suggestion
        <kirby-segmented-chip-control [items]="items" (chipSelect)="onChipSelect($event)"
```

File: src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.ts [hunk 38]
```diff
@@ -0,0 +1,16 @@
...
+
+@Component({
+  selector: 'kirby-segmented-chip-control-example',
+  templateUrl: './segmented-chip-control-example.component.html',
+  styleUrls: ['./segmented-chip-control-example.component.scss'],
+})
+export class SegmentedChipControlExampleComponent implements OnInit {
+  constructor() {}
+  items: string[] = ['chip-1', 'chip-2', 'chip-3', 'chip-4', 'chip-5', 'chip-6'];
+  selectedItem: string = 'chip-1';
```
Comment (jakobe) [inline src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.ts]:
```suggestion
  selectedItem: string = items[0];
```

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html [hunk 39]
```diff
@@ -0,0 +1,5 @@
+<div class="scrollable">
+    <ng-container *ngFor="let item of items">
```
Comment (jakobe) [inline src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html]: Remove redundant wrapper:
```suggestion
     <kirby-chip *ngFor="let item of items">
```

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html [hunk 40]
```diff
@@ -0,0 +1,7 @@
+<ScrollView orientation="horizontal">
+    <WrapLayout orientation="horizontal">
```
Comment (jakobe) [inline src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]: Are you sure you want to wrap the items??

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html [hunk 41]
```diff
@@ -0,0 +1,7 @@
...
+        <StackLayout *ngFor="let item of items" (tap)="segmentedChipSelect(item)">
```
Comment (jakobe) [inline src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]: Remove redundant wrapper:
```suggestion
        <kirby-chip *ngFor="let item of items" (tap)="segmentedChipSelect(item)" ...>
```

File: src/kirby/components/chip/chip.component.ts [hunk 42]
```diff
@@ -0,0 +1,14 @@
...
+
+@Component({
+  selector: 'kirby-chip',
+  templateUrl: './chip.component.html',
+  styleUrls: ['./chip.component.scss'],
+})
+export class ChipComponent {
+  @Input() text: string;
+  @Input()
+  @HostBinding('class.isSelected')
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.ts]: Use css naming convention:
```suggestion
  @HostBinding('class.is-selected')
```

File: src/kirby/components/chip/chip.component.scss [hunk 43]
```diff
@@ -0,0 +1,17 @@
...
+  white-space:nowrap;
+  padding: 0 25px;
+  height: 30px;
+  font-size: font-size('s');
+  line-height: line-height('xl');
+  border-radius: 25px;
+  margin-right: 5px;
+  min-width: $min-chip-width;
+  cursor: pointer;
+  &:hover,&.isSelected {
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.scss]: Use css naming convention:
```suggestion
  &:hover,&.is-selected {
```

File: src/kirby/components/chip/chip.component.tns.html [hunk 44]
```diff
@@ -0,0 +1,3 @@
+<StackLayout class="chip" [class.selected]="isSelected">
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.tns.html]: Keep naming on sync:
```suggestion
<StackLayout class="chip" [class.is-selected]="isSelected">
```

File: src/kirby/components/chip/chip.component.tns.scss [hunk 45]
```diff
@@ -0,0 +1,12 @@
+@import '../../scss/native-imports';
+$min-chip-width: 50;
+.chip {
+  border-radius: 25;
+  margin-right: 10;
+  padding:10 15;
+  font-size: font-size('xs');
+  min-width: $min-chip-width;
+  &.selected {
```
Comment (jakobe) [inline src/kirby/components/chip/chip.component.tns.scss]: Keep naming on sync:
```suggestion
  &.is-selected {
```

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.ts [hunk 46]
```diff
@@ -0,0 +1,23 @@
...
+  @Input() items: string[];
+  @Input() selectedItem: string;
+  @Output() chipSelect = new EventEmitter<string>();
+
+  constructor() {}
+  segmentedChipSelect(item: string): void {
+    this.selectedItem = item;
+    this.chipSelect.emit(item);
+  }
+  /**
```
Comment (jakobe) [inline src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.ts]: Remove comment...

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html (hunk 40 above)
Comment (gags88) [reply src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]: We have tried removing wrap layout and then output is not as expected. Can we please know reason of not using it?

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html (hunk 41 above)
Comment (gags88) [reply src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]: Redudant wrapper removed

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.ts (hunk 46 above)
Comment (gags88) [reply src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.ts]: Comment removed

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html (hunk 39 above)
Comment (gags88) [reply src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.html]: Template updated

File: src/kirby/components/chip/chip.component.ts (hunk 42 above)
Comment (gags88) [reply src/kirby/components/chip/chip.component.ts]: Done

File: src/kirby/components/chip/chip.component.tns.scss (hunk 45 above)
Comment (gags88) [reply src/kirby/components/chip/chip.component.tns.scss]: Done

File: src/kirby/components/chip/chip.component.tns.html (hunk 44 above)
Comment (gags88) [reply src/kirby/components/chip/chip.component.tns.html]: Done

File: src/kirby/components/chip/chip.component.scss (hunk 43 above)
Comment (gags88) [reply src/kirby/components/chip/chip.component.scss]: Done

File: src/app/showcase/segmented-chip-control-showcase/segmented-chip-control-showcase.component.html (hunk 35 above)
Comment (gags88) [reply src/app/showcase/segmented-chip-control-showcase/segmented-chip-control-showcase.component.html]: Done

File: src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.ts (hunk 38 above)
Comment (gags88) [reply src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.ts]: Done

File: src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.tns.html (hunk 37 above)
Comment (gags88) [reply src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.tns.html]: Done

File: src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.html (hunk 36 above)
Comment (gags88) [reply src/app/examples/segmented-chip-control-example/segmented-chip-control-example.component.html]: Done

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html (hunk 40 above)
Comment (gags88) [reply src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]: Adjusted as suggested

File: src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html (hunk 41 above)
Comment (gags88) [reply src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html]: We do not need this as it won't help as per Slack conversation.

**Pull request merged**